packages/paste-core/utilities/grid/src/helpers.tsx

```diff
@@ -26,6 +26,21 @@ export const getOuterGutterPull = (theme: ThemeShape, gutter?: Space): Margin =>
   return 'auto';
 };
 
+// Gets the gutter and returns the value to be used as padding for Columns
+export const getColumnGutters = (theme: ThemeShape, gutter?: Space): Padding => {
+  if (Array.isArray(gutter)) {
+    return (gutter as SpaceOptions[]).map((value: SpaceOptions) => {
+      return `${theme.space[value]}` as SpaceOptions;
+    });
+  }
+
+  if (gutter) {
+    return `${theme.space[gutter as SpaceOptions]}` as SpaceOptions;
+  }
+
+  return 'space0';
```

Comment:
How does this work? When there is a gutter you return the _value_ from the theme object. But here for the default, you're returning a token name.

Reply:
Oh right. That's wrong. Changed in my latest push. Returning the first value of space in the theme object now.